inspirehep/dojson/hep/fields/bd20x24x.py

```diff
@@ -24,111 +24,71 @@
 
 from __future__ import absolute_import, division, print_function
 
+import langdetect
+
 from dojson import utils
 
 from ..model import hep, hep2marc
 
 from inspirehep.utils.helpers import force_force_list
 
 
-@hep.over('title_translations', '^242[10_][0_]')
-@hep.over('titles_old', '^247[10_][10_]')
-@utils.for_each_value
-@utils.filter_values
-def title_translations_old(self, key, value):
-    """Translation of Title by Cataloging Agency."""
-    return {
-        'source': value.get('9'),
-        'title': value.get('a'),
-        'subtitle': value.get('b')
-    }
-
-
-@hep2marc.over('210', '^title_variations$')
-@hep2marc.over('242', '^title_translations$')
-@hep2marc.over('247', '^titles_old$')
-@utils.for_each_value
-@utils.filter_values
-def title_trans_var_old2marc(self, key, value):
-    """Translation of Title by Cataloging Agency."""
-    return {
-        'a': value.get('title'),
-        'b': value.get('subtitle'),
-        '9': value.get('source'),
-    }
-
-
-@hep.over('title_variations', '^210[10_][0_]')
-def title_variations(self, key, value):
-    """Title variations."""
-    def get_value(existing):
-        if not isinstance(value, (tuple, list)):
-            values = [value]
+@hep.over('titles', '^(210|24[2567])[10_][0_]')
+def titles(self, key, value):
+    """All sorts of titles"""
```

Comment:
We should start killing docstrings in DoJSON rules because they are either incorrect or useless.